src/expr/src/relation/mod.rs

```diff
@@ -3203,6 +3204,30 @@ impl RustType<proto_window_frame::ProtoWindowFrameBound> for WindowFrameBound {
     }
 }
 
+/// Maximum iterations for a LetRec.
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
+pub struct LetRecLimit {
+    /// Maximum number of iterations to evaluate.
+    pub max_iters: NonZeroU64,
```

Comment:
Because of the `min_max` computation in some of the transforms the ordering of the fields is important. Maybe worthwhile noting this here as a comment in case somebody decides to reorder this. This will also be harder to mess up if we have the `min_max` computation method next to this struct definition as suggested below.

Reply:
Actually, the `.min()` call is after extracting `max_iters`, so the field order doesn't affect it.